twitter_sanity/scraping/basic_tweet_scraping.py

```diff
@@ -45,3 +47,11 @@ def get_likes(username):
     c.Output = "fav.csv"
     likes = twint.run.Favorites(c)
     return likes
+
+# Testing
+if __name__ == '__main__':
+   username = 'elonmusk'
+   list_of_following = get_following(username)
+   followingtweets = following_tweets(list_of_following)
+   retweets = get_retweets(username)
+   likes = get_likes(username)
```

Comment:
You've added the tests, but you've clearly not tested it locally, because this file won't run.